tap_mavenlink/streams/base.py

```diff
@@ -61,11 +79,20 @@ def sync_paginated(self, params, extra):
             total_pages = result['meta']['page_count']
             LOGGER.info('Synced page {} of {} for {}'.format(page_number, total_pages, self.TABLE))
 
+            self.save_state(transformed[-1])
+
             if page_number >= total_pages:
                 break
             else:
                 params['page'] += 1
 
+    def save_state(self, last_record):
+        # TODO is this overkill since we know we don't have compisite keys here?
```

Comment:
> What would be overkill in this comment? The check whether the replication_key is in the last_record?

Good point! This TODO is no longer relevant (since we explicitly prevent composite replication-keys as part of arg validation). 

I'll remove the TODO.

Reply:
:+1: